content/en/docs/concepts/packages/_index.md

```diff
@@ -144,12 +143,13 @@ for instance:
 
     cue eval ./mypkg
 
-it will only look files with such a clause and ignore files without it.
+it will only look at files with such a clause and ignore files without it.
 
-If the package name within the directory is not unique, `cue` needs to
-know the name of the package as well
+If the package name within the directory is not unique, `cue` needs to know
```

Comment:
Again, could be left for a later PR, but I don't think this wording is not actually fully precise. The distinction is whether the package name matches the `path.Basename` of the containing directory:

```
exec cue eval
stdout 'A: "A"'
stdout 'Y: "Y"'

exec cue eval mod.com/a
stdout 'A: "A"'

exec cue eval mod.com/a:y
stdout 'Y: "Y"'

-- a/a.cue --
package a

A: "A"
-- a/y.cue --
package y

Y: "Y"
-- cue.mod/module.cue --
module: "mod.com"
-- x.cue --
package x

import (
	"mod.com/a"
	"mod.com/a:y"
)

A: a.A
Y: y.Y
```